tmt/steps/discover/fmf.py

```diff
@@ -403,23 +398,40 @@ def assert_git_url(plan_name: Optional[str] = None) -> None:
 
             # Check what should be extracted from the sources
             if dist_git_extract:
-                if dist_git_extract == '/':
-                    dist_git_extract = sourcedir
+                if len(dist_git_extract) == 1 and dist_git_extract[0] == '/':
```

Comment:
After using `normalize_path_list()`, `dist_git_extract` would be a `List[Path]`. Comparing its element to string directly wouldn't work, `dist_git_extract[0].resolve() == '/'` should be the right way.